invenio_theme/assets/semantic-ui/less/invenio_theme/theme/theme.less

```diff
@@ -0,0 +1,67 @@
+/*******************************
+        Import Directives
+*******************************/
+
+/*------------------
+       Theme
+-------------------*/
+
+@theme: @@element;
+
+/*--------------------
+   Site Variables
+---------------------*/
+
+/* Default site.variables */
+@import '@{themesFolder}/default/globals/site.variables';
+
+/* Packaged site.variables */
+@import '@{themesFolder}/@{siteFolder}/globals/site.variables';
+
+/* Component's site.variables */
+@import (optional) '@{themesFolder}/@{theme}/globals/site.variables';
+
+/* invenio-theme site.variables */
+@import (optional) 'themes/invenio/globals/site.variables';
+
+/* Site theme site.variables */
+@import (optional) '@{siteFolder}/globals/site.variables';
+
+/*-------------------
+ Component Variables
+---------------------*/
+
+/* Default */
+@import '@{themesFolder}/default/@{type}s/@{element}.variables';
+
+/* Packaged Theme */
+@import (optional, multiple) '@{themesFolder}/@{theme}/@{type}s/@{element}.variables';
+
+/* Invenio-theme Theme */
+@import (optional, multiple) 'themes/invenio/@{type}s/@{element}.variables';
```

Comment:
Question: Is this order correct?
I am wondering if the "Packaged theme" should go after the "invenio-theme theme" so that it can be overriden by the packages? (or am I missunderstanding ?)